sys/vfs_util/vfs_util.c

```diff
@@ -171,6 +171,17 @@ int vfs_is_dir(const char *path)
     return ((stat.st_mode & S_IFMT) == S_IFDIR);
 }
 
+bool vfs_file_exists(const char *path)
+{
+    int res = vfs_open(path, O_RDONLY, 0);
```

Comment:
I learned in a filesystem lecture that many programs call `stat` to check file existence.
I assume it is more light weight. It could be that `open` even calls it.

Reply:
I wasn't sure if all fs backends would implement stat, open should be always available.